client/src/shared/components/Collapse/Collapse.tsx

```diff
@@ -0,0 +1,73 @@
+import { useState } from 'react';
+
+export interface collapseProps {
+  label: string
+  subLabel?: string
+  children: React.ReactNode
+};
+
+const Collapse: React.FC<collapseProps> = ({
+  label = 'Install Xampp',
+  subLabel = 'Version 1 of 20',
```

Comment:
- instead of defining directly the default props in here you can use this

```suggestion
Collapse.defaultProps = {
  label: ''
  subLabel: ''
};
```

- Refer to other components how it is utilized